### ISSUE TITLE
Aggregate data of a cluster

### ISSUE THREAD

**Issue by lloesche** [netdata/netdata#359]
I saw the tv.html and how it visualises data from two nodes on the same page. But is there any way I can merge charts from multiple nodes on the client side? Let's say I have a 10 node web cluster and would like to visualise the SUM(req)/s in a single chart. Is that currently possible?

**Comment by ktsaou** [comment]
Hi, no currently it is not. I could add such a function to `dashboard.js`, but it has to wait a bit... too many requests, too little time...

**Comment by afletch** [comment]
This would be great to see. A graph-heavy page can be confusing and the ability to aggregate data on one graph would make for a much better representation.

**Comment by damageboy** [comment]
Agreed with the previous requests, the ability to aggregate (sum and other options) considerably simplifies monitoring

**Comment by ktsaou** [comment]
> Agreed with the previous requests, the ability to aggregate (sum and other options) considerably simplifies monitoring

Could you please provide a few use cases that such aggregations will considerably simplify monitoring?

**Comment by lloesche** [comment]
Some use cases:
- I have a Mesos cluster and would like to graph CPU allocation vs. CPU usage across the cluster
- I have a Web cluster and would like to see the total number of requests/sec
- I have a distributed database and would like to see queries/sec performance over the entire cluster.
- I have a sharded database and would like to graph my entire table space not just the portion held on each node
- I have a monitoring system storing a cluster's memory usage in a timeseries database. I would like to have a view of the memory consumption of my entire cluster instead of looking at individual nodes.

Basically any distributed/clustered application could benefit from an aggregated view. Some applications already provide such aggregated statistics (like a distributed filesystem surely shows cluster wide statistics) but many don't.

**Comment by jmlero** [comment]
Also I agree with this request.

**Comment by vincus** [comment]
- nginxes behind LVS, total requests / sec / cluster
- nginx_log python.d module, total 4xx requests / cluster
- appserver clusters, total CPU usage (SUM(ALL_NODES_CPU_USAGES)/SUM(ALL_NODES_100PERCENT)

High level monitoring clusters on a simple dashboard would be an effective way and we should buy less 60" TVs :)

**Comment by ktsaou** [comment]
You are right. It is needed.

**Comment by vincus** [comment]
Is it on the roadmap then or need any additional info ?
Thank you

**Comment by ktsaou** [comment]
@gmosx 	this is on our roadmap.

**Comment by cakrit** [comment]
Duplicate of #416 - Essentially dynamic dashboards.
